encoding/thrift/doc.go

```diff
@@ -34,11 +34,12 @@
 //
 // In addition to generating code for types specified in your Thrift file,
 // this will generate two packages for each service in the file: a client
-// package and a server package.
+// package, a server package, and a test package.
 //
 // 	myservice
 // 	   |- myserviceclient
 // 	   |- myserviceserver
+// 	   |- myservicetest
```

Comment:
Calling it `${service}test` allows for putting the server testing methods in
the same package. See original note:

> Note that the client mocks are named MockClient rather than MockInterface.
> This is to leave room for adding fake/mock servers inside the same package
> instead of creating a new package.

So the idea is that `${service}test` contains testing utilities for that
service. Right now, this only includes a client mock but in the future it can
include other things. I wanted to avoid polluting the namespace too much with
unreadable package names.